pkg/forklift-api/webhooks/mutating-webhook/mutators/secret-mutator.go

```diff
@@ -24,10 +24,12 @@ var log = logging.WithName("mutator")
 
 const NewLine = 0x0a
 
-type OvirtCertMutator struct {
+var secretChanged = false
```

Comment:
why having a global variable? it can be defined within the `Mutate` function, right?

Reply:
I initially tried to separate everything into function to make it more easy to read but it just added more complexity, forgot to return the var back inside.